onnxruntime/python/tools/tensorrt/perf/symbolic_shape_infer.py

```diff
@@ -0,0 +1,1301 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
why make a copy of symbolic_shape_infer.py script?
the source of truth is at https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/symbolic_shape_infer.py
we wouldn't want to maintain multiple copies and have them go out of sync with each other.

Reply:
I will modify the code to use symbolic_shape_infer.py only from https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/symbolic_shape_infer.py